src/NzbDrone.Core/Notifications/MailGun/MailGunSettings.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Net;
+using System.Runtime.CompilerServices;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.Notifications.MailGun
+{
+    public class MailGunSettingsValidator : AbstractValidator<MailGunSettings>
+    {
+        public MailGunSettingsValidator()
+        {
+            RuleFor(c => c.ApiKey).NotEmpty();
+            RuleFor(c => c.From).NotEmpty();
+            RuleFor(c => c.Recipients).NotEmpty();
+        }
+    }
+    
+    public class MailGunSettings : IProviderConfig
+    {
+      private static readonly  MailGunSettingsValidator Validator = new MailGunSettingsValidator();
+
+      public MailGunSettings()
+      {
+          BaseUrlUs = "https://api.mailgun.net/v3";
```

Comment:
These shouldn't be settings unless a user can actually change them. Make them constants inside the `MailGunProxy`.